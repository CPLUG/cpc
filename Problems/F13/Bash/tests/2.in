8
authorized_keys2
config
id_dsa
id_dsa.pub
id_rsa
id_rsa.pub
known_hosts
known_hosts.old
