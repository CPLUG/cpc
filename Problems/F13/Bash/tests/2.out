{authorized_keys2,config,id_{dsa{,.pub},rsa{,.pub}},known_hosts{,.old}}
